TARGETS

```diff
@@ -971,12 +489,12 @@ for test_cfg in ROCKS_TESTS:
 
 custom_unittest(
     name = "make_rocksdbjavastatic",
-    command = ["internal_repo_rocksdb/make_rocksdbjavastatic.sh"],
     type = "simple",
+    command = ["internal_repo_rocksdb/make_rocksdbjavastatic.sh"],
 )
 
 custom_unittest(
     name = "make_rocksdb_lite_release",
-    command = ["internal_repo_rocksdb/make_rocksdb_lite_release.sh"],
     type = "simple",
+    command = ["internal_repo_rocksdb/make_rocksdb_lite_release.sh"],
```

Comment:
@IslamAbdelRahman running "python buckifier/buckify_rocksdb.py" generates big change to TARGETS file. Is it expected?